capa/features/extractors/dnfile/extractor.py

```diff
@@ -30,7 +30,7 @@
     get_dotnet_unmanaged_imports,
     get_dotnet_managed_method_bodies,
 )
-
+from capa.features.extractors.strings import DEFAULT_STRING_LENGTH
```

Comment:
where does this get used?

Reply:
Changed constructor to take the value as a default.